README.md

```diff
@@ -131,14 +110,14 @@ DataPointType dataPoint = new DataPointType() { Timestamp = DateTime.UtcNow, Val
 OmfDataMessage dataMessage = OmfMessageCreator.CreateDataMessage(streamId, dataPoint);
 ```
 
-Then the devices uses its own ingress client, which uses the device clientId and clientSecret to authenticate the requests. The device clientId is used to route the message to the Topic that the clientId is mapped to. Note that the message must be serialized before being sent.
+Then the devices uses its own ingress client, which uses the device clientId and clientSecret to authenticate the requests. The device clientId is used to route the message to the Connection that the clientId is mapped to. Note that the message must be serialized before being sent.
```

Comment:
The note seems out of plan.  I realize that this is legacy, but I feel like that is a different topic than the previous sentances

Reply:
i meant that the "Note that the message must be serialized before being sent." seems like it should be handled different.  It felt lost  in the information

Reply:
I see what you mean now. I removed the Note